grids/graphdb.go

```diff
@@ -31,24 +31,38 @@ func (kgraph *GDB) DeleteGraph(graph string) error {
 
 	gkey := kgraph.keyMap.GetGraphKey(graph)
 
+	var anyError error
```

Comment:
Could you use `"github.com/hashicorp/go-multierror"` here? It is important for a user to know which vertices/edges failed get deleted.